worker/detectors/packages/pacman.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015, 2016 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package packages defines PackagesDetector for several sources.
+package packages
+
+import (
+	"bufio"
+	"strings"
+
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/utils"
+	"github.com/coreos/clair/utils/types"
+	"github.com/coreos/clair/worker/detectors"
+)
+
+// PacmanPackagesDetector implements PackagesDetector and detects pacman packages
```

Comment:
All docs should be full sentences, end with periods, and be wrapped for 80-characters lines.